src/components/Form.jsx

```diff
@@ -26,98 +28,167 @@ export default function Form() {
+      </Card>
+
+      <Card isVisible={visibleCard === 'sizeAndPriceInput'}>
+        <label htmlFor="Size">
+          4. Size of the product <span>(optional)</span>
+        </label>
+        <input
+          onChange={(event) => setSize(event.target.value)}
+          value={size}
+          maxLength="10"
+          id="size"
+          placeholder="50 ml"
+        />
+        {size.length >= 10 && (
+          <p>The product size can consist of up to 10 characters.</p>
+        )}
+
+        <label htmlFor="Price">
+          5. Price of the product <span>(optional)</span>
+        </label>
         <input
-          onChange={(event) => setMonth(event.target.value)}
-          value={month}
-          type="number"
-          min="1"
-          max="120"
-          id="month"
-          placeholder="E.g. 12"
+          onChange={(event) => setPrice(event.target.value)}
+          value={price}
+          maxLength="10"
+          id="price"
+          placeholder="12€"
         />
-        <InfoPopover />
-      </ContainerStyled>
-      {month > 120 && (
-        <p>The product can expire up to 120 months after opening.</p>
-      )}
-
-      <label htmlFor="Size">
-        4. Size of the product <span>(optional)</span>
-      </label>
-      <input
-        onChange={(event) => setSize(event.target.value)}
-        value={size}
-        maxLength="10"
-        id="size"
-        placeholder="50 ml"
-      />
-      {size.length >= 10 && (
-        <p>The product size can consist of up to 10 characters.</p>
-      )}
-
-      <label htmlFor="Price">
-        5. Price of the product <span>(optional)</span>
-      </label>
-      <input
-        onChange={(event) => setPrice(event.target.value)}
-        value={price}
-        maxLength="10"
-        id="price"
-        placeholder="12€"
-      />
-      {price.length >= 10 && (
-        <p>The product price can consist of up to 10 characters.</p>
-      )}
-
-      <Button
-        text="Save"
-        disabled={!(name && date && month)}
-        className="save-button"
-      />
+        {price.length >= 10 && (
+          <p>The product price can consist of up to 10 characters.</p>
+        )}
+        <div className="button-container">
+          <Button
+            text="Back"
+            isCancel
+            onClick={() => setVisibleCard('monthInput')}
+          />
+          <Button
+            text="Save"
+            type="submit"
+            disabled={!(name && date && month)}
+            onClick={handleSubmit}
+          />
+        </div>
+        <div className="step-indicator">
+          <Circle/>
+          <Circle/>
+          <Circle/>
+          <Circle className="active"/>
+        </div>
+      </Card>
+      <Card isVisible={visibleCard === 'success'}>
+        <h2>The product has been successfully saved.</h2>
+        <i className="far fa-check-circle" aria-hidden="true"/>
+        <Button text="Close" type="button" onClick={handleCloseClick}/>
```

Comment:
Nach dem man das Popover schließt, landet man in der Mobilansicht (Safari & Chrome) irgendwo in der Mitte der Seite und muss dann zum Produkt hochscrollen. Vielleicht kannst du da ja noch ein scroll to top einbauen. 
🙂
https://stackoverflow.com/questions/33188994/scroll-to-the-top-of-the-page-after-render-in-react-js

Reply:
Gute Idee! 👩‍💻